common-rest/src/main/java/uk/ac/ebi/uniprot/api/rest/output/converter/AbstractEntityIdHttpMessageConverter.java

```diff
@@ -1,9 +1,10 @@
 package uk.ac.ebi.uniprot.api.rest.output.converter;
 
 import org.springframework.http.MediaType;
-
+import org.springframework.lang.Nullable;
```

Comment:
Do we want this or an accident?

Reply:
I am overriding the method exactly how is used by spring. I think I will keep it.